src/rpc/core_rpc_server.cpp

```diff
@@ -1312,6 +1312,7 @@ namespace cryptonote
     response.depth = m_core.get_current_blockchain_height() - height - 1;
     response.hash = string_tools::pod_to_hex(hash);
     response.difficulty = m_core.get_blockchain_storage().block_difficulty(height);
+    response.cumulative_difficulty = response.block_weight = m_core.get_blockchain_storage().get_db().get_block_cumulative_difficulty(height);
```

Comment:
This part ` = response.block_weight = ` seems like a mistake (when copying a line?), although harmless as it's written two lines below.